### ISSUE TITLE
Do you consider to release the training code?

### ISSUE THREAD

**Issue by happycoding1996** [drilistbox/CWD#2]
Hi, thanks for your work!

Could you release the training code? My implementation cannot reach the performance reported in your paper.

Thanks in advance!

**Comment by drilistbox** [comment]
> Hi, thanks for your work!
> 
> Could you release the training code? My implementation cannot reach the performance reported in your paper.
> 
> Thanks in advance!

Sorry to reply you later. I am a little busy recently, and the code is not ready yet. 
1. code for ablation will come soon.
2. code for seg: https://github.com/pppppM/mmsegmentation/tree/laoyong_cvpr
3. code for detection: https://github.com/pppppM/mmdetection

===================================================
run this code:【https://drive.google.com/file/d/15BdezuKf4pdDwk9KhhEeCLkS1CmCdxya/view?usp=sharing】
The mIou (no flip and scale) of original student(PSPNet-RN18) is 71.2
./tools/dist_train_distill.sh ./configs/pspnet/distill-pspnet_r101-to-pspnet_r18-d8_512x1024_40k_cityscape-ocw-0.0-v1.py None 1 45680

After distillation only by CW loss, the mIou (no flip and scale) of original student(PSPNet-RN18) is 74.7.
./tools/dist_train_distill.sh ./configs/pspnet/distill-pspnet_r101-to-pspnet_r18-d8_512x1024_40k_cityscape-ocw-40.0-v1.py None 1 45682

**Comment by happycoding1996** [comment]
@drilistbox Thanks for your prompt reply and experimental details!

**Comment by qingliucs** [comment]
Hi, have you reproduced the training phase? Is it possible to share your code?

**Comment by drilistbox** [comment]
hi, thanks for your attention, more detail please refer https://github.com/irfanICMLL/TorchDistiller/tree/main/SemSeg-distill.

**Comment by qingliucs** [comment]
> hi, thanks for your attention, more detail please refer https://github.com/irfanICMLL/TorchDistiller/tree/main/SemSeg-distill.

Thanks a lot.
